src/main/java/com/example/notes/service/NoteService.java

```diff
@@ -23,4 +23,10 @@ public NoteDtoList getAllNotes() {
         return new NoteDtoList(noteDtos);
     }
 
+    public NoteDto createNote(NoteDto noteDto) {
```

Comment:
**createNote** can be renamed to **create**.

Reply:
Done